.github/workflows/ci.yml

```diff
@@ -226,9 +226,18 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-# this occupies 8084 port on Ubuntu 20.04 and we don't use it.
-      - name: Remove mono blocking 8084 port
-        run: sudo kill -9 $(sudo lsof -t -i:8084)
+      - name: Stop Mono and XSP
+        # These occupy port 8084 unnecessarily
+        run: |
+          sudo systemctl stop mono-xsp4.service || true
+          sudo systemctl disable mono-xsp4.service || true
+          sudo killall mono || true
+          sudo killall xsp4 || true
```

Comment:
Nobody can survive that :smile:

Reply:
> Nobody can survive that 😄

:-D It's really piece of crap design. I want to make sure that if we simply kill without disabling, there won't be a process that will bring it up again.

Reply:
I remember I tried to uninstall it but that did not work (but I don't remember details anymore).

Reply:
I guess that's because uninstall would just disable the service via systemctl which doesn't stop the process blocking the socket..